### PR TITLE
fix: TT-178-change cancelation message

### PR DESCRIPTION
**Pull request by safeamiiir**
- change cancelation message
- use CircularProgress component instead of loading text

### CODE REVIEW

**Review by str-anger — APPROVED** [review]

File: aqueductcore/frontend/src/__tests__/TaskDetailsModal.test.tsx [hunk 1]
```diff
@@ -53,7 +53,7 @@ test("clicking on cancel task button cancels task", async () => {
     await userEvent.click(cancelTaskConfirmationButton);
 
     await waitFor(() => {
-        const cancelTaskConfirmationMessage = queryByText("Task cancelled successfully");
+        const cancelTaskConfirmationMessage = queryByText("Cancelation request sent");
```
Comment (jatinriverlane) [inline aqueductcore/frontend/src/__tests__/TaskDetailsModal.test.tsx:56]: This message seems to be fine for Querra, which includes sending a cancellation request to some other API, but for the internal tasks, this message does not seem to be right.

Maybe we can keep this message for now and ask Tom for an updated message when he's back.

File: aqueductcore/frontend/src/components/organisms/TaskDetailsModal/index.tsx [hunk 2]
```diff
@@ -289,7 +289,7 @@ function TaskDetailsModal({ isOpen, handleClose, taskId }: TaskDetailsModalProps
                             </List>
                         </Grid>
                         <Grid item>
-                            {isTaskCancellable ? loadingCancelTask ? <div>loading</div> : <CancelTaskButton
+                            {isTaskCancellable ? loadingCancelTask ? <CircularProgress sx={{ mr: 4 }} /> : <CancelTaskButton
```
Comment (jatinriverlane) [inline aqueductcore/frontend/src/components/organisms/TaskDetailsModal/index.tsx:292]: The size of CircularProgress bar appears to be much bigger compared to the size of the cancellation button. How about decreasing the size of the circular progress bar ?

Comment (safeamiiir) [reply aqueductcore/frontend/src/components/organisms/TaskDetailsModal/index.tsx]: i changed its size into 32px

File: aqueductcore/frontend/src/__tests__/TaskDetailsModal.test.tsx (hunk 1 above)
Comment (safeamiiir) [reply aqueductcore/frontend/src/__tests__/TaskDetailsModal.test.tsx:56]: this is the ultimate information that we could receive in the response, how can I tell the cancelation has been successful or not? for example could you tell me a combination of returnCode or Status or etc?

Comment (jatinriverlane) [reply aqueductcore/frontend/src/__tests__/TaskDetailsModal.test.tsx:56]: Makes sense. In future, if there's any hard requirement we can come up with a further procedure of checking with server if the task is completely cancelled or not. 

It's fine for now.

**Pull request merged**